components/infra-proxy-service/cmd/infra-proxy-service/commands/serve.go

```diff
@@ -97,7 +99,8 @@ func serve(cmd *cobra.Command, args []string) {
 	// get secrets client
 	secretsClient := secrets.NewSecretsServiceClient(secretsConn)
 
-	service, err := service.Start(l, migrationConfig, connFactory, secretsClient, authzClient)
+	service, err := service.Start(l, migrationConfig, connFactory, secretsClient, authzClient, authzClientProject)
```

Comment:
`authzProjectClient` can be a better name of the variable?
```suggestion
	service, err := service.Start(l, migrationConfig, connFactory, secretsClient, authzClient, authzProjectClient)
```

Reply:
Changed to the suggested name